docs/INSTALL.md

```diff
@@ -141,5 +143,5 @@ kubectl delete  -f https://raw.githubusercontent.com/confidential-containers/ope
 
 ### Delete the Operator
 ```
-kubectl delete -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl delete -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```

Comment:
Same comment here.

Reply:
I take this back, this is not that simple to do without assuming users have some specific tools installed on their OSes.